packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -67,6 +67,11 @@ enum ScrollDecelerationRate {
 /// // ...
 /// final ScrollPhysics mergedPhysics = physics.applyTo(const AlwaysScrollableScrollPhysics());
 /// ```
+///
+/// When implementing a subclass, you must override [applyTo] so that it returns
+/// an appropriate instance of your subclass.  Otherwise, classes like
+/// [Scrollable] that consume a [ScrollPosition] may lose your customizations
+/// when they attempt to combine them with a default [ScrollPhysics] object.
```

Comment:
```suggestion
/// [Scrollable] that inform a [ScrollPosition] will combine them with the default [ScrollPhysics] object instead of your custom subclass.
```